Solutions/Marain.TenantManagement.Specs/Features/TenantManagement/CreateClientTenant.feature

```diff
@@ -19,3 +19,24 @@ Scenario: Create a new client tenant with a well known GUID
 	Given the tenancy provider has been initialised for use with Marain
 	When I use the tenant management service to create a new client tenant with well known Guid 'd620a845-de25-4b88-b5a1-27d6b1104f82' called 'Litware'
 	Then the tenant with Id '75b9261673c2714681f14c97bc0439fb45a820d625de884bb5a127d6b1104f82' is called 'Litware'
+
+Scenario: Create a new child client tenant of a client tenant with a well known GUID 
+	Given the tenancy provider has been initialised for use with Marain
+	And I have an existing client tenant with a well known Guid '102fdab0-6d9c-4f99-bf87-ce94ad3bbae6' called 'Fabrikam'
+	When I use the tenant management service to create a new child client tenant of the 'Fabrikam' client tenant with well known Guid 'd620a845-de25-4b88-b5a1-27d6b1104f82' called 'Litware'
+	Then the tenant with Id '75b9261673c2714681f14c97bc0439fbb0da2f109c6d994fbf87ce94ad3bbae645a820d625de884bb5a127d6b1104f82' is called 'Litware'
+	And the tenant with Id '75b9261673c2714681f14c97bc0439fbb0da2f109c6d994fbf87ce94ad3bbae645a820d625de884bb5a127d6b1104f82' has a parent with Id '75b9261673c2714681f14c97bc0439fbb0da2f109c6d994fbf87ce94ad3bbae6'
+
+Scenario: Create a new child client tenant of a client tenant which does not exist 
+	Given the tenancy provider has been initialised for use with Marain
+	And the client tenant 'Fabrikam' does not exist
+	When I use the tenant management service to create a new child client tenant of the 'Fabrikam' client tenant with well known Guid 'd620a845-de25-4b88-b5a1-27d6b1104f82' called 'Litware'
+	Then an 'TenantNotFoundException' is thrown
```

Comment:
Super-picky: this should be *a*, not *an*.

Reply:
Copy-paste fail. Fixed